ispo-working-group-charter.md

```diff
@@ -1,41 +1,45 @@
 # ISPO Working Group Charter
 
-The purpose of this working group is to support InnerSource Program Offices (ISPOs) with materials needed to establish themselves, teach and scale InnerSource in their respective company.
-This material could be directed towards senior leaders, managers, or individuals in a variety of roles.
+This working group aims to provide materials to InnerSource Program Offices (ISPOs) to help them establish themselves, teach InnerSource, and scale. Senior leaders, managers, or individuals in various roles can benefit from this material.  
 
 ## Summary
 
-We believe that most training and teaching content can be shared across companies when those institutions share the same relevant points of context.
-While not every company is exactly the same, not every company is wholly different, either.
-For each challenge in InnerSource, we believe that there are a small number of ways of common ways to address it that should work for most companies.
-Our goal is to document these sets of solutions to challenges, make it easy for ISPOs to find which apply to them, and then to produce sharable, open source implementations of these solutions.
-These implementations will likely be documents, videos, or code, each of which is generic enough to be used across companies,
-yet specific enough that people in the company can leverage them to solve their problems without needing notable amounts of company-specific work.
+The majority of training and teaching content can be shared across companies when the contexts are similar.  
```

Comment:
A big 👍 on this!

I'm aware of a demand for an InnerSource Whitepaper for executives. A template to generate a polished internal Whitepaper for companies would be epic!

Is there already a pattern for a generic InnerSource working model? I didn't find one here: 

https://github.com/InnerSourceCommons/InnerSourcePatterns

e.g.

```mermaid
flowchart LR;
    SCM-->Search;
    Search-->Repository;
    Repository-->Development;
    Development-. "Promote" .-> Contribution;
    Contribution-. "Promote" .-> Repository;
```

Reply:
> In the end, the desired outcomes and goals to sell on depend strongly on the organizations current challenges and desires. As does for this matter the focus on various aspects of working mechanisms.

This sounds like an opportunity to define clear content principles if they aren't already defined somewhere; I presume they will be part of future conversations.

Reply:
Agree.